CHANGELOG.rst

```diff
@@ -1,5 +1,12 @@
 CHANGELOG for molecule
 ======================
+1.1.1
+----
+
+* Cleaned up state file management logic to be more concise, functional for other purposes.
+* Removed --fast flag from converge in favor of using state file for fast converging.
+* Instance hostname is now printed during serverspec runs.
+
```

Comment:
I would prefer we not release this yet, until we fix up the ansible.cfg template bug, so others can override the template with a path to an alternate template.  As it stands today, if we release this, we will break our ansible-systems testing.